### PR TITLE
Added products resource for subscriptions app - (Pre Prod)

### PR DESCRIPTION
**Pull request by ssaurav-redhat**
**This MR is aimed at adding new permissions in ci/stage/qa/prod**

- Read/Manage Products

**It will also add role changes into ci/stage/prod/qa:**

- Modifying "Subscriptions user" role to have read access to products resource

### CODE REVIEW

**Comment by coderbydesign** [comment]
@ssaurav-redhat to test these in pre-production environments before the changes move to prod, please break this into two PRs, one for pre-prod environment, and once validated, one for prod. Let us know if you have any questions!

**Comment by ssaurav-redhat** [comment]
Hey @coderbydesign ,
I have made changes in the PR as mentioned. Please review and let me know if any other change is required for this.

File: configs/qa/roles/subscriptions.json [hunk 1]
```diff
@@ -32,6 +32,9 @@
         {
           "permission": "subscriptions:organization:read"
         },
+        {
```
Comment (coderbydesign) [inline configs/qa/roles/subscriptions.json:35]: You'll need to bump the `"version"` above, to `3` in all environment files for the seed changes to be picked up.

Comment (ssaurav-redhat) [reply configs/qa/roles/subscriptions.json:35]: I have made the required changes. Please review

**Review by coderbydesign — APPROVED** [review]

**Pull request merged**